pkgs/by-name/ox/ox/package.nix

```
@@ -34,7 +34,7 @@ rustPlatform.buildRustPackage rec {
     description = "Independent Rust text editor that runs in your terminal";
     homepage = "https://github.com/curlpipe/ox";
     changelog = "https://github.com/curlpipe/ox/releases/tag/${version}";
-    license = licenses.gpl2Only;
+    license = licenses.gpl2Plus;
```

```suggestion
    license = licenses.gpl2;
```

Should I use deprecated ID for this case? https://github.com/NixOS/nixpkgs/blob/5792ff1913926ee5b57b55a0690abf292021f626/lib/licenses.nix#L1374-L1378
https://github.com/Homebrew/homebrew-core/pull/204891#issuecomment-2601854038